resources/apache.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+#
+# Cookbook:: rundeck
+# Resource:: apache
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+include RundeckCookbook::Helpers
+
+property :use_ssl, [true, false], default: false
+property :cert_name, String, default: node['hostname']
```

Comment:
You'll want to lazily eval this:

```suggestion
property :cert_name, String, default: lazy { node['hostname'] }
```